frontend/src/components/table/TableWrapper.js

```diff
@@ -156,6 +156,32 @@ class TableWrapper extends PureComponent {
     }
   };
 
+  handleFocusAction = ({ fieldName, supportFieldEdit }) => {
+    this.setState({
+      contextMenu: {
+        ...this.state.contextMenu,
+        fieldName,
+      },
+      supportFieldEdit,
+    });
+  };
+
+  handleFastInlineEdit = () => {
+    const { selected } = this.props;
+    const { contextMenu, supportFieldEdit } = this.state;
+    // a routine needs to be added in here - to check if field is allowed to be edited
+    // if (contextMenu.supportFieldEdit && selected.length === 1) {
```

Comment:
What's this comment for ? Is this `routine` something we need now ? In the future ?

Reply:
Is just from an initial attempt to restrict using the shortcut. Commented out and forgot to remove the comment. Removed now